oops/hosts/cassini/iss.py

```diff
@@ -43,17 +43,19 @@ def from_file(filespec, fast_distortion=True,
     vicar_dict = vic.as_dict()
 
     # Get key information from the header
-    tstart = julian.tdb_from_tai(julian.tai_from_iso(vic['START_TIME']))
+    tstop = julian.tdb_from_tai(julian.tai_from_iso(vic['IMAGE_TIME']))
     texp = max(1.e-3, vicar_dict['EXPOSURE_DURATION']) / 1000.
+    tstart = tstop - 0.5*texp
```

Comment:
If `IMAGE_TIME` is the stop time, why are you multiplying the exposure by 0.5 to get the start time?

Reply:
Typo.  I copied that form OMINAS, which wants the mid time.